README.md

```diff
@@ -68,7 +68,13 @@ ENV Variable | Description |
 `GITHUB_CLIENT_SECRET`| the GitHub Application Client Secret.
 `NON_STAFF_GITHUB_ADMIN_IDS` | GitHub `user_ids` of users to be granted staff level access.
 
+To find your Client ID/Secret you can register a new OAuth app at [this](https://github.com/settings.applications/new) page - after creation you should be able to see your Client ID/Secret.
```

Comment:
I'm not a really big fan of linking `this` what do you think bout

To obtain a GitHub Client ID/Secret you need to [register a new OAuth application](https://github.com/settings.applications/new).

If we are telling users how to setup the application it would probably be a good idea to tell them how to setup the URL scheme.

```
# Regular url
http://localhost:5000

# Callback url
http://localhost:5000/auth/github/callback
```